config/routes.rb

```diff
@@ -188,8 +188,10 @@
           post :payment_gateways, to: 'communities#create_payment_gateway'
           get :social_media
           get :analytics
+          get :maps
```

Comment:
To my understanding this will create a route with path like /admin/communities/253462/maps. We have decided that we don't want to expose the community id in the URL. However, some old URLs still do it.

So I'd suggest that you change this. Have a look how the route for look and feel and details have been configured: https://github.com/sharetribe/sharetribe/pull/2289/files#diff-21497849d8f00507c9c8dcaf6288b136R172